src/client/components/Tile.jsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { withStyles } from '@material-ui/core/styles';
+import Card from '@material-ui/core/Card';
+import CardContent from '@material-ui/core/CardContent';
+import CardMedia from '@material-ui/core/CardMedia';
+
+import customStyles from './Tile.css';
+
+const styles = {
+  card: {
+    height: 500,
+    maxWidth: 345,
```

Comment:
any reason for the random cutoff? looks fine but why not like 350 or something haha